rust/gui-client/build.bat

```diff
@@ -10,5 +10,8 @@ call pnpm tsc
 REM Compile CSS
 call pnpm tailwindcss -i src\input.css -o src\output.css
 
+REM bundle JS
+call pnpm vite build
```

Comment:
I am surprised `vite` doesn't do the tailwind stuff for you?

Reply:
It might. I don't remember exactly why I added vite other than that I want to match the sample Tauri code. So it probably has lots of features I'm not familiar with yet.